frontend/src/app/components/scan-viewer/scan-viewer.component.ts

```diff
@@ -71,7 +71,9 @@ export class ScanViewerComponent implements OnInit, AfterViewInit {
     }
 
     public sliderFocus() {
-        this.slider._elementRef.nativeElement.focus();
+        window.setTimeout(() => {
+            this.slider._elementRef.nativeElement.focus();
+        }, 10);
```

Comment:
Looks pretty hacky. How about that timeout value, why exacly `10` ?

Reply:
>Under IE, an element that is not visible cannot receive focus. Depending on the version of IE, it doesn't always re-render (or re-flow) the document until after the current JavaScript function has returned. This leads to all kinds of weird behavior, but mostly, you don't see visual updates during JavaScript code blocks. Combined with the inability to assign focus, that line was being silently ignored.

>Adding the setTimeout effectively gives control back to the browser, so it can reflow the document. Then, it immediately runs the timer function, and sets the focus. The 10ms timeout is enough, because the browser will take over even if there's a timer waiting.`

(quote from stackoverflow ;))

So, essentially, timeout length does not matter, what matters is solely the presence of timeout.